File: abc/abc262/a.java
// https://atcoder.jp/contests/abc262/submissions/41452091

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner stdIn = new Scanner(System.in);

    int y = stdIn.nextInt();
    if(y % 4 == 2) {
      System.out.println(y);
    } else {
      System.out.println(y + (4 - (y - 2) % 4));
    }
  }
}

File: abc/abc262/a.cs
// https://atcoder.jp/contests/abc262/submissions/46505734

using System;

public class A
{
  static void Main()
  {
    int y = int.Parse(Console.ReadLine());
    for(int i = y; i <= 5000; i++)
    {
      if(i % 4 == 2)
      {
        Console.Write(i);
        return;
      }
    }
  }
}

